src/Core/Core.js

```diff
@@ -3886,6 +3889,7 @@ export default class Core {
     stateVarObj.getPreviousDependencyValuesForMarkStale = arrayStateVarObj.getPreviousDependencyValuesForMarkStale;
```

Comment:
Do you want to add docs as well above on line 3880, and point people to the new docs in BaseComponent.

Reply:
OK